db_scripts/mosip_idpbinding/dml.sql

```diff
@@ -2,6 +2,7 @@
 
 ----- TRUNCATE idpbinding.public_key_registry TABLE Data and It's reference Data and COPY Data from CSV file -----
 TRUNCATE TABLE idpbinding.public_key_registry cascade ;
+TRUNCATE TABLE idpbinding.id_token_mapping cascade ;
```

Comment:
Its DML script, we should not truncate idpbinding.public_key_registry and idpbinding.id_token_mapping.
Instead we should truncate idpbinding.key_policy_def table.

Reply:
Fixed